arch/posix/include/posix_soc_if.h

```diff
@@ -3,8 +3,8 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#ifndef ZEPHYR_ARCH_POSIX_INCLUDE_POSIX_SOC_IF_H_
-#define ZEPHYR_ARCH_POSIX_INCLUDE_POSIX_SOC_IF_H_
+#ifndef ZEPHYR_INCLUDE_ARCH_POSIX_POSIX_SOC_IF_H_
```

Comment:
Nit: If the header does not move, the include guard does not need changing.

Reply:
That seems to have slipped in while moving posix_soc_if.h back into public. I will fix that.